galaxychop/core.py

```diff
@@ -507,27 +457,26 @@ def jcirc(self, bin0=0.05, bin1=0.005):
         Parameters
         ----------
         bin0 : float, default=0.05
-            Size of the energy bin of the inner part of the galaxy, in the
-            range of (-1, -0.1) of the normalized energy.
-
+            Size of the specific energy bin of the inner part of the galaxy,
+            in the range of (-1, -0.1) of the normalized energy.
         bin1 : float, default=0.005
-            Size of the energy bin of the outer part of the galaxy, in the
-            range of (-0.1, 0) of the normalized energy.
+            Size of the specific energy bin of the outer part of the galaxy,
+            in the range of (-0.1, 0) of the normalized energy.
 
         Returns
         -------
         gx : galaxy object
-            new instanced galaxy with x (normalized energy) and
-            y (z component of the normalized angular momentum).
+            new instanced galaxy with x (normalized specific energy) and
+            y (z component of the normalized specific angular momentum).
             See section Notes for more details.
 
         Notes
         -----
         The x and y values are calculated from the binning in the normalized
-        energy. It is selected in each bin the particle with the highest
-        value of z component of standardized angular momentum and is assigned
-        its standardized energy value to x and its component value z angular
-        momentum normalized to y.
+        specific energy. It is selected in each bin the particle with the
```

Comment:
"In each bin, the particle with the highest value of z component of standardized specific angular momentumis selected"*